public/app/core/components/NestedFolderPicker/NestedFolderList.tsx

```diff
@@ -111,25 +113,36 @@ function Row({ index, style: virtualStyles, data }: RowProps) {
   }
 
   return (
-    <div style={virtualStyles} className={styles.row}>
-      <input
-        className={styles.radio}
-        type="radio"
-        value={id}
-        id={id}
-        name="folder"
-        checked={item.uid === selectedFolder}
-        onChange={handleRadioChange}
-        onKeyDown={handleKeyDown}
-      />
-
+    // don't need a key handler here, it's handled at the input level in NestedFolderPicker
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events
+    <div
+      ref={rowRef}
+      style={virtualStyles}
+      className={cx(styles.row, {
+        [styles.rowFocused]: index === focusedItemIndex,
+        [styles.rowSelected]: item.uid === selectedFolder,
+      })}
+      tabIndex={-1}
+      onClick={handleSelect}
+      aria-expanded={isOpen}
+      aria-selected={item.uid === selectedFolder}
+      aria-labelledby={labelId}
+      aria-level={level + 1} // aria-level is 1-indexed
+      role="treeitem"
+      aria-owns={children.length > 0 ? children.map((child) => getDOMId(child)).join(' ') : undefined}
```

Comment:
this is ugly but i don't think there's another way 😬 i was hoping there was a way to specify a parent from the child, but looking through the spec it looks like each parent must explicitly list all it's children 🤮